x/stake/errors.go

```diff
@@ -16,6 +16,7 @@ const (
 	CodeInvalidValidator CodeType = 201
 	CodeInvalidBond      CodeType = 202
 	CodeInvalidInput     CodeType = 203
+	CodeValidatorJailed  CodeType = 204
```

Comment:
Jailed? Revoked? - what does jailed mean?

Reply:
oh I get it now - yeah maybe more comments throughout explaining "jailed" - reference in the spec would be güd too